internal/testrunner/runners/system/runner.go

```diff
@@ -1118,7 +1120,10 @@ func (r *runner) setupService(ctx context.Context, config *testConfig, serviceOp
 	}
 
 	serviceDeployer, err := servicedeployer.Factory(serviceOptions)
-	if err != nil {
+	if errors.Is(err, os.ErrNotExist) {
+		return nil, svcInfo, nil
+	}
+	if err != nil && !errors.Is(err, os.ErrNotExist) {
```

Comment:
This change should also be applied into `internal/service/boot.go` 
https://github.com/elastic/elastic-package/blob/f8153ab029d82c2107f3c6d0f6de5a9a478550d9/internal/service/boot.go#L38

and `internal/benchrunner/runners/system/runner.go`
https://github.com/elastic/elastic-package/blob/f8153ab029d82c2107f3c6d0f6de5a9a478550d9/internal/benchrunner/runners/system/runner.go#L251

Reply:
Done in both places, but not really tested in benchmarks, I will address issues there if any in a follow up.